configs/peer/config.json

```diff
@@ -1,6 +1,9 @@
 {
-  "PUBLIC_KEY": null,
-  "PRIVATE_KEY": null,
+  "PUBLIC_KEY": "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0",
+  "PRIVATE_KEY": {
+    "digest_function": "ed25519",
+    "payload": "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
+  },
```

Comment:
Do you think it's better this way? My understanding was that it's obviously OK for test environments, but when trying to spin up a peer, we want to refrain from giving defaults, instead prompting the user to generate the key and provide it directly. More so for the keys, less so for `torii` and trusted peers, but still relevant.

Reply:
Agree with you both, will try serialize proxy config.